### PR TITLE
A cigarette will fall out of your mouth once it is burnt.

### LINKED ISSUE #10725: Finishing a smoke leaves the butt in your mouth slot
## Description
You should drop the butt.

**Reproduction**
Smoke a cig until it is a butt.
It remains in your mouth

### PR DESCRIPTION
**Pull request by PrPleGoo**
Fixes #10725

### CODE REVIEW

**Comment by Visne** [comment]
https://github.com/space-wizards/space-station-14/issues/10725 is already closed, for good reason.

**Comment by PrPleGoo** [comment]
For no reason at all actually

**Comment by Visne** [comment]
I have already stated my reasons and multiple people have already shared their thoughts, I am not going to write them down again here.

**Comment by tomasalves8** [comment]
Yep no from me as well.

**Comment by moonheart08** [comment]
I do not think making every spaceman ever automatically litter is good.

**Comment by PrPleGoo** [comment]
> I do not think making every spaceman ever automatically litter is good.

It is good, it gives the janitor more reason to exist.

**Comment by ike709** [comment]
Cringe closer, this was replicating the SS13 behavior

**Comment by Iamgoofball** [comment]
this is replicating the behavior from SS13, yes

**Comment by PaulRitter** [comment]
I closed bc it seems that the majority is against this. Ik it replicates ss13 behaviour, but given thats not our main focus when developing ss14, im gonna need to hear better arguments

**Comment by PaulRitter** [comment]
to add ontop of that, the issue this pr is "fixing" was already closed, hence i took this pr as obsolete

**Comment by Iamgoofball** [comment]
ok heres a better reason:
it kinda sucks to click cigarette butts in your inventory slots and i foresee it being annoying for players to not realize the 2px icon in their mask slot is blocking them wearing a new mask

**Comment by Iamgoofball** [comment]
however better solutions if we keep this is to make the cigarette butt sprite larger and easier to click

**Comment by Visne** [comment]
In my opinion clicking inside an inventory slot anywhere should be the same as just clicking the item in it.

**Comment by Emisse** [comment]
pretty sure it is

**Comment by Visne** [comment]
Yeah, I just checked and it is. SS13 issue !!!!

**Comment by moonheart08** [comment]
> ok heres a better reason: it kinda sucks to click cigarette butts in your inventory slots and i foresee it being annoying for players to not realize the 2px icon in their mask slot is blocking them wearing a new mask

tl;dr, non-issue. I assume it's safe for this to stay closed considering clicking anywhere in the slot works?

**Comment by PrPleGoo** [comment]
I don't care about clicking on the slot, that works fine in SS14.
It's a feature I expected and, in my opinion, is part of the smoking abstraction.
The following is at least party motivated reasoning, but I think it still merits the change.

### Real life parallels
When you smoke a cig in real life you take drags.
Once you get close to the filter you stop smoking and extinguish it either on the floor or an ash tray.
If you were to keep taking drags I'm pretty sure you would start inhaling burnt filter smoke.

In space station (13 and 14) you just continuously inhale small amounts.
In 13 you drop the extinguished butt once you "finish" the cig.
Looking at the sprite you can tell it still has a little left, like a butt would in real life.
To me this is an abstraction of chucking the still lit cig on the floor and stomping it out.

In 14 the cig just extinguishes itself.
I don't think anyone wants to see cigs be implemented in such a way that the smoker needs to grab and toss the still lit cig to prevent smoking plastic at the end.
If this is the desired level of abstraction, then go ahead and make an issue for that - I would personally not support this.

### Gameplay
The cig butt might be a piece of evidence left behind for the detective to find.
If you drop it on a crime scene you can become a suspect, opening new avenues for roleplay.

The cig butt is a piece of litter for the janitor to clean up.

Lighting the next cig is two fewer clicks (click one is grabbing the cig, click two is dropping the cig).

**Comment by PaulRitter** [comment]
im gonna do whats called a gamer move

**Comment by PaulRitter** [comment]
all of the reasons listed above actual make alot of sense to me, imo having this has more enrichment to rp than the alternative

**Comment by PrPleGoo** [comment]
Awesome! Now we can start discussing the changes :^)

**Comment by Visne** [comment]
As a player that cleans up their own trash, I really hate that it would drop on the ground only for me to have to pick it up right away again.

Suggestion: if you have a hand free your cigarette goes into your hand, otherwise it drops on the ground.

If you want to talk about realism, I think this makes the most sense: if you have a hand free, you get it in your hand so you can then decide what to do with it (throw it in the trash or on the ground), and if you have your hands full because you're busy, you spit it on the ground.

**Comment by PaulRitter** [comment]
> As a player that cleans up their own trash, I really hate that it would drop on the ground only for me to have to pick it up right away again.
> 
> Suggestion: if you have a hand free your cigarette goes into your hand, otherwise it drops on the ground.
> 
> If you want to talk about realism, I think this makes the most sense: if you have a hand free, you get it in your hand so you can then decide what to do with it (throw it in the trash or on the ground), and if you have your hands full because you're busy, you spit it on the ground.

this sounds super janky. it would basically suddenly block a hand slot with minimal prior warning. this would annoy the shit out of me more than picking up a dropped cig butt would ever

**Comment by PrPleGoo** [comment]
> As a player that cleans up their own trash

I think you are a minority of a minority.
You can still unequip it prematurely and dispose of it as you like.

**Comment by Visne** [comment]
> > As a player that cleans up their own trash
> 
> I think you are a minority of a minority.
> You can still unequip it prematurely and dispose of it as you like.

I'm not throwing away a lit cigarette that I have no idea off how long it will still burn.

**Comment by PrPleGoo** [comment]
You can wait for it to go out.

**Comment by Visne** [comment]
> You can wait for it to go out.

What? When it goes out it will be on the ground.

**Comment by PrPleGoo** [comment]
Not if its in your hand

**Comment by Visne** [comment]
I'm not ~~throwing away~~ holding a lit cigarette that I have no idea off how long it will still burn.

**Comment by PrPleGoo** [comment]
Why not?

**Comment by Visne** [comment]
Because that's weird?

**Comment by PrPleGoo** [comment]
Okay champ

**Comment by Zumorica** [comment]
This is far from a constructive argument for something that's honestly insignificant. I'm gonna make a vote live on lizard and see what the 64 players in there prefer

**Comment by Zumorica** [comment]
The people have decided.
image

**Comment by PaulRitter** [comment]
democracy and its consequences have been a disaster for this project

**Comment by HoidC** [comment]
> democracy and its consequences have been a disaster for this project

ok well quite simply if you want it enough just merge it using your mind like that octo guy did

channel your inner junior contributor
